resources/assets/coffee/react/beatmap-discussions-history/main.coffee

```diff
@@ -90,34 +91,35 @@ export class Main extends React.PureComponent
 
 
   render: =>
-    el DiscussionsContext.Provider,
-      value: @discussions()
-      el BeatmapsContext.Provider,
-        value: @beatmaps()
-        div className: 'modding-profile-list modding-profile-list--index',
-          if @props.discussions.length == 0
-            div className: 'modding-profile-list__empty', osu.trans('beatmap_discussions.index.none_found')
-          else
-            for discussion in @props.discussions
-              div
-                className: 'modding-profile-list__row'
-                key: discussion.id,
-
-                a
-                  className: 'modding-profile-list__thumbnail'
-                  href: BeatmapDiscussionHelper.url(discussion: discussion),
-
-                  img className: 'beatmapset-cover', src: discussion.beatmapset.covers.list
-
-                el Discussion,
-                  discussion: discussion
-                  users: @users()
-                  currentUser: currentUser
-                  beatmapset: discussion.beatmapset
-                  isTimelineVisible: false
-                  visible: false
-                  showDeleted: true
-                  preview: true
+    el ReviewEditorConfigContext.Provider, value: @props.reviewsConfig,
```

Comment:
Should probably at least be consistent with the other contexts below (i.e. `value: ...` on the following line, or changing them to match)